README.md

```diff
@@ -61,36 +64,72 @@ for instance for `HH+HV` add both `HH` and `HV`.
 **Important:** In the `properties` of a STAC Item `sar:polarizations` must be a set with unique elements. 
 In assets `sar:polarizations` can contain duplicate elements and, if possible, the polarizations must appear in the same order as in the file.
 
+### Beam Object
+
+The beam object is used to describe the swath of the SAR acquisition.
+According to the instrument mode defined in `sar:instrument_mode`, the `sar:swath` is composed of one or more beams.
+
+| Field Name     | Type                  | Description                                                                            |
+| -------------- | --------------------- | -------------------------------------------------------------------------------------- |
+| id             | string                | **REQUIRED.** The unique identifier of the beam. It can be the subswath identifier     |
+| relative_burst | number                | Identification number that uniquely identifies a burst cycle within each repeat cycle. |
+| polarizations  | \[string]             | Any combination of polarizations.                                                      |
+| view:*         | View extension fields | Additional fields from the [View extension](https://github.com/stac-extensions/view)   |
```

Comment:
I think in other extensions we just added text for such cases. The Type is also not quite a type... ;-)

```suggestion

It is recommended to enrich the Beam Object fields with fields from the [View extension](https://github.com/stac-extensions/view), e.g. the incidence angle.
```